.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: go
 go:
-  #- 1.5  go test arguments are different in 1.5 :-(
-  - 1.6
```

Comment:
If our program doesn't run with a version - then you are right - lets get it out of here